content/docs/details/backend-reading-allow-list/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Accessing Public resources from the Backend
+publishedDate: '2021-04-26T21:00:00.0Z'
+lastUpdated: '2022-04-26T08:33:00.0Z'
+description: The Backstage backend restricts what resources can be fetched by default. This describes how to add sources which can be accessed.
+---
+
+## Introduction
+
+The Backstage backend restricts what resources can be fetched by default. Only urls matching `backend.reading.allow` in 
```

Comment:
```suggestion
The Backstage backend restricts what resources can be fetched by default. 
```

Remove the internal Backstage config knowledge. I wouldn't expect customers to know anything about this.